config/sync/field.field.node.stanford_event.su_event_location.yml

```diff
@@ -0,0 +1,47 @@
+uuid: 9fb70bde-f581-48c4-95ff-da1ef05f429e
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.su_event_location
+    - node.type.stanford_event
+  module:
+    - address
+id: node.stanford_event.su_event_location
+field_name: su_event_location
+entity_type: node
+bundle: stanford_event
+label: Location
+description: 'Where the event is taking place.'
+required: false
+translatable: false
+default_value: {  }
```

Comment:
Yeah, I avoided it for just that reason. It could be worth the effort, however. I tried to default to a Stanford Address as well but that also caused fun with the importer.